content/microservices-and-eliminating-implicit-trust.md

```diff
@@ -7,9 +7,8 @@ category: "Runtime environment"
 The perimeter for containerized applications deployed as microservices is the microservice itself. Therefore, it is necessary to define policies that restrict communication only between sanctioned microservice pairs. The inclusion of zero trust in the microservice architecture reduces the blast radius by preventing lateral movement should a microservice be compromised. Operators should ensure that they are using capabilities such as network policies to ensure that east-west network communication within the container deployment is limited to only that which is authorized for access. There is some initial work done to provide strategies for microservices security through NIST SP 800-204 and may serve as a guide for implementing secure microservice architectures.
```

Comment:
```suggestion
The perimeter for containerized applications deployed as microservices is the microservice itself. Therefore, it is necessary to define policies that restrict communication only between sanctioned microservice pairs. The inclusion of zero trust in the microservice architecture reduces the blast radius by preventing lateral movement should a microservice be compromised. Operators should ensure that they are using capabilities such as network policies to ensure that east-west network communication within the container deployment is limited to only that which is authorized for access. There is some initial work done to provide strategies for microservices security through [NIST SP 800-204](https://doi.org/10.6028/NIST.SP.800-204) and may serve as a guide for implementing secure microservice architectures.
```

Reply:
also - should this bulk text be 80 char break?  or will this effect the rendering in map?

Reply:
shouldn't be, I think at the end of it we will run a linter on the entire thing and do it all in one PR